nion/swift/Inspector.py

```diff
@@ -2080,13 +2137,14 @@ class ImageDisplayInspectorSection(InspectorSection):
 
     def __init__(self, document_controller: DocumentController.DocumentController, display_item: DisplayItem.DisplayItem) -> None:
         super().__init__(document_controller.ui, "display-limits", _("Image Display"))
-
```

Comment:
Both `_scalebar_options_handler` and `_image_diaplay_handler` should be private; and as long as you're in this code, I'd be ok fixing the spelling of `_image_diaplay_handler`. Also let's standardize on "scale marker", two words, rather than "scalebar" one word.

```
self.__scale_marker_options_handler = ...
self.__image_display_handler = ...
```

As I said above, I'm open to another name than "scale marker" but let's use the same terms throughout the code.